.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,71 @@
+name: Build wheel
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [windows-latest, ubuntu-latest, macos-latest, macos-14]
+    env:
+      CIBW_SKIP: 'pp*'
+      CIBW_ARCHS: 'auto64'
+      CIBW_PROJECT_REQUIRES_PYTHON: '>=3.9'
+      CIBW_TEST_REQUIRES: 'pytest'
+      CIBW_TEST_COMMAND: 'pytest {project}/tmp/tests'
```

Comment:
Why is it "{project}/tmp/tests" given that the tests directory is in the main project directory?

Reply:
This appends the project directory path since build is not done in the project directory